.github/workflows/es-actions.yml

```diff
@@ -49,29 +49,39 @@ jobs:
     - uses: actions/checkout@v3
       with:
         submodules: true
-    - name: Setup tmate session
-      uses: mxschmitt/action-tmate@v3
-      timeout-minutes: 15
-      with:
-        detached: true
+    - uses: lukka/get-cmake@latest
     - uses: GuillaumeFalourd/setup-windows10-sdk-action@v1.11
       with:
         sdk-version: 20348
     - uses: ilammy/msvc-dev-cmd@v1.12.1
       with:
         arch: x86
         sdk: "10.0.20348.0"
-    - uses: lukka/get-cmake@latest
-    - name: Build Win32 Release
+    - name: Install msvc redist package
       run: |
         (new-object System.Net.WebClient).DownloadFile('https://github.com/abbodi1406/vcredist/releases/download/v0.73.0/VisualCppRedist_AIO_x86_x64.exe','VisualCppRedist_AIO_x86_x64.exe')
         .\VisualCppRedist_AIO_x86_x64.exe /y
+    - name: Copy octane
+      run: |
+        copy test\octane\*.js
+        dir
+    - name: Build Win32 Release
+      run: |
         CMake -G "Visual Studio 16 2019" -DCMAKE_SYSTEM_NAME=Windows -DCMAKE_SYSTEM_VERSION:STRING="10.0" -DCMAKE_SYSTEM_PROCESSOR=x86 -DESCARGOT_ARCH=x86 -DESCARGOT_MODE=release -Bout/win32_release_ninja/ -DESCARGOT_HOST=windows -DESCARGOT_OUTPUT=shell -DESCARGOT_LIBICU_SUPPORT=ON -DESCARGOT_LIBICU_SUPPORT_WITH_DLOPEN=OFF -DESCARGOT_TEST=ON -G Ninja -DCMAKE_C_COMPILER=cl -DCMAKE_CXX_COMPILER=cl -DCMAKE_BUILD_TYPE=release
-        cd out/win32_release_ninja
-        ninja
-        copy ..\..\test\octane\*.js
-        .\escargot.exe run.js
-
+        CMake --build out/win32_release_ninja/ --config Release
+        .\out\win32_release_ninja\escargot.exe run.js
+    - uses: ilammy/msvc-dev-cmd@v1.12.1
+      with:
+        arch: x64
```

Comment:
Just a question)
Is it possible to switch to x64 from x86 in the same action job?

Reply:
Yes. ilammy/msvc-dev-cmd support switch every arch to arch on-the-fly
I will use matrix for parallel build